docs/python/python-quick-start.md

```diff
@@ -0,0 +1,125 @@
+
+![Gif showing Code Actions in a Python project.](images/editing/quickFix.gif)
+
+## Python Commands
+
+Python commands can be accessed through the [Command Palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette) (`kb(workbench.action.showCommands)`). From the Command Palette, you have access to the full functionality of VS Code, but for Python specific functionality begin typing **“Python: “** to locate the commands accessible through the Python extension.
+
+![Gif demonstrating how to access Python commands in the Command Palette.](/docs/python/images/quick-start/cmd-plt-python.gif)
+
+## Run, debug, and test
+
+Now that you are more familiar with Python in VS Code, let’s learn how to run, debug, and test your code.
+
+### Run
+
+There are a few ways to run Python code in VS Code.
+
+To run the active Python file, click the **Run Python File in Terminal** play button in the top-right side of the editor.
+
+![Image showing the Run Python File in Terminal play button.](images/tutorial/run-python-file-in-terminal-button.png)
+
+There are three other ways you can run Python code within VS Code:
+-	Right-click anywhere in the editor window and select **Run > Python File** in Terminal (which saves the file automatically):
+![Image showing the Python run options in the context menu.](images/tutorial/run-python-file-in-terminal.png)
+-	Select one or more lines, then press `kbstyle(Shift+Enter)` or right-click and select **Run Selection/Line in Python Terminal**. This command is convenient for testing just a part of a file.
+-	From the Command Palette (`kb(workbench.action.showCommands)`), select the **Python: Start REPL** command to open a REPL terminal for the currently selected Python interpreter. In the REPL, you can then enter and run lines of code one at a time.
+
+### Debug
+
+Debugging starts by setting breakpoints in your Python project. This will allow you to explore data as it moves through your program. You can set a breakpoint by clicking in the gutter of the line, where a red dot will appear.
+
+To start debugging, initialize the debugger by pressing `F5`. Since this is your first time debugging this file, a configuration menu will open from the Command Palette allowing you to select the type of debug configuration you would like for the opened file or project.
+
+The program will stop at the breakpoint you set, and you can track data in the Python Debug console, and progress through your program using the debug toolbar.
+
+![Gif showing how to configure the Python debugger for the first time.](images/quick-start/qs-python-debug.gif)
+
+After selecting a debug configuration, you can also start the debugger by clicking on the down-arrow next to the run button on the editor and selecting **Debug Python File in Terminal**.
```

Comment:
I'd suggest to remove this too, keeping it simple with F5 (or direct users to the Run and Debug view instead. Either or, we should probably suggest one flow only)